apps/app_sla.c

```diff
@@ -434,7 +434,7 @@ static const char *trunkstate2str(enum sla_trunk_state state)
 	S(SLA_TRUNK_STATE_ONHOLD)
 	S(SLA_TRUNK_STATE_ONHOLD_BYME)
 	}
-	return "Uknown State";
```

Comment:
app_sla.c doesn't exist in the 20 branch.  You'll have to remove `cherry-pick-to: 20` from this PR and submit a separate PR just for the 20 branch with a `cherry-pick-to: none` comment.

Reply:
In this PR, is this enough for removing 20 https://github.com/asterisk/asterisk/pull/1193#issuecomment-2789846711

Or do i need to make a new comment that only includes 21 and 22 ?